daemon/rename.go

```diff
@@ -9,7 +9,7 @@ import (
 // reserved.
 func (daemon *Daemon) ContainerRename(oldName, newName string) error {
 	if oldName == "" || newName == "" {
-		return fmt.Errorf("usage: docker rename OLD_NAME NEW_NAME")
+		return fmt.Errorf("Neither old nor new names may be empty")
```

Comment:
@cpuguy83 I'm a bit nervous about changing this because we return 500 in other cases where the user gave us bogus data and I'd prefer to change all of them at once instead of just this one.